utils/utils.go

```diff
@@ -77,3 +79,25 @@ func ToISO8601DateTimeString(dateTime time.Time) string {
 		"%04d-%02d-%02dT%02d:%02d:%02d.%03dZ",
 		year, month, day, hour, minute, second, micros)
 }
+
+// add env-vars from annotations.
+// see https://github.com/dapr/dapr/issues/2508.
+func ParseEnvString(envStr string) []corev1.EnvVar {
+	envVars := make([]corev1.EnvVar, 0)
```

Comment:
Super small (not even worth mentioning) nit: If you put this below the `envPairs` split then you can initialize it with a capacity and possibly avoid an allocation. In general, it's [best practice better to pass capacity](https://medium.com/swlh/bad-go-not-sizing-slices-aed1b01cff83) into `make` if you know it. That said, there are probably so few env vars in this scenario that it would not help. I believe the default capacity for a slice is `5`.

```go
envVars := make([]corev1.EnvVar, 0, len(envPairs))
```